docs/README.md

```diff
@@ -0,0 +1,87 @@
+# pepdbagent
+
+`pepdbagent` is a Python package for uploading, updating, and retrieving PEP metadata from a Postgres database.
+
+Generally pepdbagent consists of its core - **PEPDatabaseAgent** class that conduct 3 main parts: **Projects**, 
+**Projects Annotations**, **Namespace Annotations**. These parts have different purpose and were separated
+to increase readability, maintainability and simplicity of pepdbagent package. Below, you can find 
+more detailed information about them.
+
+## PEPDatabaseAgent
+PEPDatabaseAgent is core function, it helps set connection to database (by using **BaseConnection** class)
+and provide user with 3 main modules: `project`,  `annotation`, `namespace`.
+
+Example:
+Set connection to database
+```python
+
+import pepdbagent
+# 1) By providing credentials and connection information:
+agent = pepdbagent.PEPDatabaseAgent(user="postgres", password="docker", )
+# 2) or By providing connection string:
+agent = pepdbagent.PEPDatabaseAgent("postgresql://postgres:docker@localhost:5432/pep-db")
```

Comment:
You probably meant: `postgresql://docker@localhost:5432/pep-db` right?

Reply:
Nope, everything is correct, but I forgot to add `dsn=...`

Reply:
i see now. The username being `postgres` threw me off...